src/dashboard/src/main/migrations/0038_fix_workflow_store_dip.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+"""Migration to change workflow related to DIP upload and storage."""
+
+from __future__ import print_function, unicode_literals
+
+from django.db import migrations
+
+
+def data_migration(apps, schema_editor):
+    """This migration modifies the workflow related to uploading and storing
+    of DIPs. The end result is the sub-workflow schematized below:
+
+                                    Upload DIP?
+                                       /  \
+                                     Yes   No
+                                    /      |
+                              Atom, etc.   |
+                                 /         |
+                            move to        |
+                            uploadedDIPs/  |
+                                         \ |
+                                      Store DIP?
+                                         /  \
+                                       Yes   No
+                                       |     |
+                               Store on SS   If uploaded, do nothing
+                                             If NOT uploaded, move to rejected
+
+    The workflow above is better for the following reaons:
+
+    1. There is only one "Store DIP Location" choice, so it can more easily
+       be made a preconfigured choice within the current system
+    2. It is more clear that when a DIP is neither uploaded nor stored, it is
+       rejected.
+
+    Steps:
+
+    1. Remove "Reject DIP" from options at "Upload DIP" decision point:
+
+    2. Rename the "Store DIP" chain's description to "Do Not Upload DIP" and
+       set its starting link to the "Store DIP?" question
+
+    3. Remove the unused links and their dependencies:
+
+       - "Retrieve DIP Storage Locations" (ed5d8475-3793-4fb0-a8df-94bd79b26a4c)
+       - "Store DIP location" (b7a83da6-ed5a-47f7-a643-1e9f9f46e364)
+       - "Store DIP" (e85a01f1-4061-4049-8922-5694b25c23a2)
+       - "Move to the uploadedDIPs directory"
+         (e3efab02-1860-42dd-a46c-25601251b930)
+
+    4. Change "Completed" link to "Handle unstored DIP". This is handled by a
+       new client script that moves the DIP to the rejected/ directory iff it
```

Comment:
Joel explained to me that this is the way that biconditionals are abbreviated in English - makes sense! :)